fitbenchmarking/results_processing/plots.py

```diff
@@ -45,6 +44,26 @@ def __init__(self, best_result, options, figures_dir):
 
         self.figures_dir = figures_dir
 
+    @staticmethod
+    def write_html_with_link_plotlyjs(fig, figures_dir, htmlfile):
+        """
+        Writes an html file for the figure passed as input and
+        includes link to the relevant plotly.js file.
+
+        :param fig: Figure to be saved
+        :type fig: plotly.graph_objs._figure.Figure
+        :param figures_dir: The directory to save the figures in
+        :type figures_dir: str
+        :param htmlfile: Name of the figure
+        :type htmlfile: str
+
+        :return: None
+        """
+
+        plotly_path = '../../../js/plotly.js'
```

Comment:
Rather than hard link the path, can you use something like we do here:
https://github.com/fitbenchmarking/fitbenchmarking/blob/4757fb1b0fa3ba0c664632d9cf646a2439f5ff31/fitbenchmarking/utils/misc.py#L66C40-L66C82
(maybe even calling that file?)

I can't think of any cases where what you've done will fail, but this protects us from needing to hunt and change lots of hard coded paths if we ever change where the js files live in the future.